Rollbar/RollbarClient.cs

```diff
@@ -23,19 +27,22 @@ public RollbarClient(RollbarConfig config)
             Config = config;
         }
 
-        public RollbarResponse PostAsJson(Payload payload)
+        public RollbarResponse PostAsJson(Payload payload, IEnumerable<string> scrubFileds)
         {
             Assumption.AssertNotNull(payload, nameof(payload));
 
             var jsonData = JsonConvert.SerializeObject(payload);
+            jsonData = JsonScrubber.ScrubJson(jsonData, scrubFileds);
```

Comment:
I would have this call a different method than the generic `ScrubJson` like `ScrubJsonPayload` which can pull out the data field, scrub that, and then put the scrubbed data back together with the Rollbar access token.

Reply:
done.